src/release/rollups.md

```diff
@@ -0,0 +1,60 @@
+# Rollup Procedure
+
+## Background
+
+The Rust project has a policy that every pull request must be tested after merge
+before it can be pushed to master. As PR volume increases this can scale poorly,
+especially given the long (>2hr) current CI duration for Rust.
+
+Enter rollups - low risk changes (often doc fixes or other non-functional
+changes) are marked with the `rollup` command to bors (`@bors r+ rollup` to
```

Comment:
Plenty of functional changes are rolled up (and marked with `rollup`) -- but they are often *small*, are not *perf sensitive*, and *not platform dependent*.